content/en/apps/examples/supervisor-reference-app.md

```diff
@@ -59,17 +59,20 @@ With the supervisor reference app, a supervisor can schedule a quality monitorin
 |**1b Supervisor UHC mode** Using the UHC mode, the supervisor is able to identify CHWs that have not been visited by a supervisor for a while| | **3b Shadowing reminder task** - the task will appear on the CHW app two days before the visit to remind the CHW of the planned shadowing visit|
 | | | **3c Quality monitoring form** The quality monitoring form can also be accessed as an action on the profile of the CHW|
 | | | **3d Quality Monitoring Follow up task** Just in case a supervisor identifies some weak areas that a CHW need to improve on and be supported, they schedule for a quality monitoring follow up visit|
-### Group session workflow
+
+### Group sessions
 
 | CHW Group session workflow |
 |--|--|
 |{{< figure src="img1_1.png" class="right" >}} | {{< figure src="img3.png" >}}|
 |**1a A CHW organizes for a group session with community members** Group session is a meeting coordinated and organized by CHW with community members to health educate community members of various health topics | **2a Group session form** Supervisor attends the group session meetings, provides supervision, supports the activity and submits the group session form|
 
-### CHW Activity workflow
+### CHW Inactivity
```

Comment:
Is it CHW activity?

Reply:
Ya, in a way it is... however, a common workflow for performance management is to identify _inactive_ CHWs that need follow-up. Calling it _CHW inactivity_ explicitly makes it clearer, and with the language often used by partners.